public/scripts/app.js

```diff
@@ -47,66 +39,36 @@ $('ul.nav li').click(function(){
          `
        );
      }
-   }
-   else if(($(this).context.textContent)==="My Work")
-   {
-     //projects section
-
+    } else if(($(this).context.textContent)==="My Work") {
      $("#addProject").fadeIn();
      $("#addVacation").fadeOut();
-
-     console.log("Projects section");
      $.ajax({
        method:"GET",
-       url:'/api/profile/projects',
+       url:'/api/profile',
        success:function(data){
-         console.log(data);
-         $p_id=data[0]._id;
-         console.log($p_id);
-         $("#results").empty();
-
-         console.log(data[0].projects.length);
-         renderAllProjects(data);
-
+         renderAllProjects(data[0]);
        },
        error: function(err){
-         console.log("ajax call failed for My Work",err);
-       }
-     })
-     var link=`/api/profile/${$p_id}/projects/:_id`
-     $.ajax({
```

Comment:
This AJAX call 1) does nothing to the page, and 2) gives me the same exact response as `/api/profile`.